ocaml/idl/datamodel.ml

```diff
@@ -954,6 +954,18 @@ module PIF = struct
              Api_errors.pif_has_fcoe_sr_in_use]
       ()
 
+  let set_disallow_unplug = call
```

Comment:
Does "set_disallow_unplug=true" mean, that a PIF can not be unplugged? The name says "disallow" but the documentation says "whether unplugging the PIF is *allowed*". Negation always risks some confusion and I therefore would suggest: (1) to extend the API documentation as below or similar, (2) consistently talk about allowing or disallowing.

```
~doc:"Set whether unplugging the PIF is allowed. True=unplugging is prohibited, False=unplugging allowed."
```